tests/integration/Owncloud/OcisPhpSdk/ShareCreatedModifyTest.php

```diff
@@ -76,17 +77,19 @@ public function testDeleteIndividualShare(): void
     public function testDeleteGroupShare(): void
     {
         $philosophyHatersGroup =  $this->ocis->createGroup(
-            'philosophyhaters',
+            'philosophy-haters',
             'philosophy haters group'
         );
         $this->createdGroups = [$philosophyHatersGroup];
         $philosophyHatersGroup->addUser($this->einstein);
 
-        $this->fileToShare->invite([$this->einstein, $philosophyHatersGroup, $this->marie], $this->viewerRole);
+        $this->fileToShare->invite($this->einstein, $this->viewerRole);
+        $this->fileToShare->invite($philosophyHatersGroup, $this->viewerRole);
+        $this->fileToShare->invite($this->marie, $this->viewerRole);
         $shares = $this->ocis->getSharedByMe();
         foreach ($shares as $share) {
             $this->assertInstanceOf(ShareCreated::class, $share);
-            if ($share->getReceiver()->getDisplayName() === 'philosophyhaters') {
+            if ($share->getReceiver()->getDisplayName() === 'philosophy-haters') {
```

Comment:
```suggestion
            if ($share->getReceiver()->getDisplayName() === 'philosophyhaters') {
```

We removed these `-` yesterday.